tests/_utils/check_status

```diff
@@ -35,7 +35,7 @@ do
 
     # if status is online, will check the max commit ts, the new max commit ts should greater than the old one.
     if [ $count -ne 1 ] || ([ $STATE == "online" ] && [ $max_commit_ts_new -le $max_commit_ts_old ]); then
-        if [ $i -eq 15 ]; then
+        if [ $i -eq 20 ]; then
```

Comment:
I tried this on my KVM and when it is set to 15 it may fail under extreme situations, i.e. `pump` restarts soon after the collector runs `updateStatus`. The time has started to be counted but the collector will update this pump's status after 5s.

Reply:
So this means after this change, it may take at least five seconds for the drainers the notice that the pump is back?

Reply:
At most. Not at least.

Reply:
the problem is drainer only refresh the status period, should better impl the watch strategy. 
But I think 5 second is acceptable for restart.

Reply:
`$i` could never equal to `20`

Reply:
I'm working on that. Label is temporarily set to **DNM**.

Reply:
Revised, PTAL again. @leoppro

Reply:
After the fix of pump quit bug now this test can be done in 15s, so we needn't change this any more. I think this conversation can be closed.